.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "endOfLine": "lf",
```

Comment:
This seems to be `lf` by default:
https://prettier.io/docs/en/options.html#end-of-line

Reply:
I needed to add this for the tests to pass on windows, looks like there's a bug in prettier

Reply:
So prettier fixes line endings as well?
Then we can remove [`lineending:production`](https://github.com/FredrikNoren/ungit/blob/6009c23e5e9cce78416f9d9ee55fcebdbb8a3920/Gruntfile.js#L424) and [`grunt.loadNpmTasks('grunt-lineending');`](https://github.com/FredrikNoren/ungit/blob/6009c23e5e9cce78416f9d9ee55fcebdbb8a3920/Gruntfile.js#L412) from the gruntfile.